### ISSUE TITLE
(Appsflyer) Package releases not tracked

### ISSUE THREAD

**Issue by mattlennon3** [segmentio/analytics-react-native#1007]
Today I wanted to look at the latest changes in `@segment/analytics-react-native-plugin-appsflyer`, as renovate has highlighted an available upgrade to the dependency.

I see that there are no changes since 0.6.0 in the releases page, and the `CHANGELOG.md` file in that section of the repo has not been updated since 0.6.0. I know there is a 0.7.0 version and the latest version is now 0.8.0.

Is there a release process that is no longer being followed? I am uncomfortable upgrading when I do not know what has changed.

Thanks

**Comment by alanjcharles** [comment]
Hi @mattlennon3 thanks for bringing this to our attention. It seems something isn't working as we would expect on our end but moving forward you should expect that changes to the appsflyer plugin will be tracked and posted in the release notes. Apologies for the inconvenience.

**Comment by mattlennon3** [comment]
Hi @alanjcharles , I have noticed the last release published in github was April 30th, but there have been releases since then. For instance a fix I needed here: https://github.com/segmentio/analytics-react-native/issues/1082#issuecomment-3026517010 was released, but I had no idea which version. 

I hope you guys are working on a more public release process, it's really important for the community to be open about what changes are contained in release versions. Thanks.
